pydsa/max_subarray.py

```diff
@@ -0,0 +1,22 @@
+#Time Complexity: O(n)
+#Algorithmic Paradigm: Dynamic Programming
```

Comment:
Kadane's algorithm have optimal structure but it do not have overlapping subproblems, i think we cant call it DP